pkg/scheduler/server/api.go

```diff
@@ -181,7 +185,9 @@ func (s *Server) triggerJob(ctx context.Context, req *api.TriggerRequest) bool {
 		// another long running go routine that accepts this job on a channel
 		log.Errorf("Error sending job to connection stream: %s", err)
 	}
+	monitoring.RecordTriggerDuration(meta.GetNamespace(), meta.GetAppId(), now)
```

Comment:
We shouldn't have app ID as a label as this produces high cardinality. Once could also say the same for namespace, though that it probably fine.

Users interested in app ID specific metrics should be getting that from daprd.

Reply:
I can rm the appID